web_ui/src/shared/components/header/jobs-management/jobs-list.component.tsx

```diff
@@ -41,11 +42,10 @@ export const JobsList = ({
     gap = 10,
 }: JobsListProps): JSX.Element => {
     const discardType: DISCARD_TYPE | undefined = JOB_STATE_TO_DISCARD_TYPE[jobState];
+    const [selected, setSelected] = useState<Selection>(new Set([]));
 
-    const handleFetchNextPage = async () => {
-        if (hasNextPage && !isFetchingNextPage) {
-            await fetchNextPage();
-        }
+    const handleFetchNextPage = () => {
+        hasNextPage && fetchNextPage();
```

Comment:
Why was this changed? Wouldn't this trigger multiple fetches at a time? 
Though I feel like ideally that should be the responsibility of `fetchNextPage` itself, so I don't mind this getting removed.

Reply:
It won’t trigger multiple times. On line 72, `isFetchingNextPage` is passed to `VirtualizedJobList`, which forwards it to `useLoadMore`. That hook ensures it only runs when the loader is not active